src/python/pants/core/goals/test.py

```diff
@@ -303,9 +303,7 @@ def register_options(cls, register) -> None:
             help=(
                 "Additional environment variables to include in test processes. "
                 "Entries are strings in the form `ENV_VAR=value` to use explicitly; or just "
-                "`ENV_VAR` to copy the value of a variable in Pants's own environment. `value` may "
-                "be a string with spaces in it such as `ENV_VAR=has some spaces`. `ENV_VAR=` sets "
-                "a variable to be the empty string."
```

Comment:
The last two sentences seem like superfluous details to me. I think most people would expect for spaces to work, right?

Reply:
Yeah, I agree.